### PR TITLE
Function.py

### PR DESCRIPTION
**Pull request by ssnskar**
Added the python code snippet of Tutorial: Using Lambda with Amazon SQS as per : https://app.asana.com/0/1203681999313912/1205789652274515

*Issue #, if available:* https://github.com/aws-samples/serverless-snippets/issues/85

*Description of changes:*
Added the python code snippet of "Tutorial: Using Lambda with Amazon SQS" to integration-sqs-to-lambda as per : https://app.asana.com/0/1203681999313912/1205789652274515

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by boyney123** [comment]
Thank you @ssnskar 

Can you please update the `snippet-data.json` file within this snippet folder please? Add this file and lang to the array inside there.

**Review by eldritchideen — COMMENTED** [review]
It would be more consistent with other examples if the `lambda_handler` function was at the top of the snippet with the `process_message` function following.

**Comment by ssnskar** [comment]
Done. Changed the position of the Lambda Handler. Kindly confirm and let me know if anything else is required.

File: integration-sqs-to-lambda/Function.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def lambda_handler(event, context):
+
```
Comment (eldritchideen) [inline integration-sqs-to-lambda/Function.py:2]: Remove this blank line.

**Comment by ssnskar** [comment]
Removed blank space.

**Review by eldritchideen — APPROVED** [review]

**Comment by boyney123** [comment]
Thanks!

**Pull request merged**